webapp/shop/decorators.py

```diff
@@ -98,11 +105,32 @@ def decorated_function(*args, **kwargs):
                     maintenance_start <= time_now < maintenance_end
                 )
 
+            if cred_maintenance_start and cred_maintenance_end:
+                _maintenance_start = parse(os.getenv("CRED_MAINTENANCE_START"))
+                _maintenance_end = parse(os.getenv("CRED_MAINTENANCE_END"))
+                _time_now = datetime.now(pytz.utc)
+                _is_in_timeframe = (
+                    _maintenance_start <= _time_now <= _maintenance_end
+                )
+                if _time_now > _maintenance_end:
```

Comment:
Here too. It would be helpful to understand if the variable names clearly show what they are for.